pysat/_custom.py

```diff
@@ -59,8 +64,8 @@ def add(self, function, kind='add', at_pos='end', *args, **kwargs):
         """Add a function to custom processing queue.
 
         .. deprecated:: 2.2.0
-          `add` will be removed in pysat 3.0.0, it is replaced by
-          `attach` to clarify the syntax
+          `Custom.add` will be removed in pysat 3.0.0, it is replaced by
```

Comment:
```suggestion
          `Custom.attach` will be removed in pysat 3.0.0, it is replaced by
```

Reply:
Except if they've accessed this warning, they've used `custom.add` and not `custom.attach` and could reasonably think we've made a mistake.  This is double deprecated, but 🐳